src/orders/src/main/java/mushop/orders/services/OrdersService.java

```diff
@@ -24,6 +24,9 @@
 import static mushop.orders.controllers.OrdersController.OrderFailedException;
 import static mushop.orders.controllers.OrdersController.PaymentDeclinedException;
 
+import mushop.orders.config.MonitoringConfiguration;
```

Comment:
![image](https://user-images.githubusercontent.com/551007/97805475-009ec800-1c1c-11eb-90f5-096930d6ce2e.png)

Reply:
ha ha :-)

Reply:
I think you meant to say monitoringClient here.